.unbroken_exclusions

```diff
@@ -0,0 +1,14 @@
+!node_modules
```

Comment:
What this file is used for?

Reply:
this lists which folders to skip checking links in.